lib/dsl/validator.js

```diff
@@ -256,7 +259,7 @@ class JDLSyntaxValidatorVisitor extends BaseJDLCSTVisitorWithDefaults {
   checkConfigPropSyntax(key, value) {
     const propertyName = key.tokenType.tokenName;
     const validation = configPropsValidations[propertyName];
-    if (_.isUndefined(validation)) {
+    if (validation === undefined) {
```

Comment:
I meant to check for undefined here, not null, empty strings, zero ...

Reply:
`!validation`

Reply:
It depends on what `configPropsValidations` returns. It's a custom function, so we know and can control what it returns.

Reply:
fixed